spec/07-effects.md

```
@@ -16,21 +16,21 @@ The effect system is based on algebraic effects with row polymorphism, inspired
 
 ```
-effect State<T> {
-  op get(Unit -> T);
-  op put(T -> Unit);
+effect Store<T> {
+  op fetch(Unit -> T);
+  op stash(T -> Unit);
 }
 ```
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)